### DISCUSSION TITLE
Is there a way to fix a sortable item in place by index while maintaining the normal sorting behavior and animation?

### DISCUSSION THREAD

**Discussion by MorkWood2** [clauderic/dnd-kit#1350] [General]
I'd like to create a sortable context with many sortable items and I was wondering the possibility for setting a specific item to be fixed so that it maintains it's index and other cards are moved around the fixed item as though it is a nominal behavior. Haven't seen an example of it, nor anything in the docs that can pull off that magic.  Curious if any users have got it working.

**Comment by little077** [comment]
no
